components/WaveHandButton.tsx

```diff
@@ -1,15 +1,24 @@
 import React from "react";
 
 interface HandButtonProps {
+  key: string;
   selected: boolean;
   onClick: () => void;
   text: string;
 }
-export function WaveHandButton({ selected, onClick, text }: HandButtonProps) {
-  return <button
-    className={`wave-hand-button ${selected ? 'selected' : ''}`}
-    onClick={onClick}
-  >
-    {text}
-  </button>;
+export function WaveHandButton({
+  key,
+  selected,
+  onClick,
+  text
+}: HandButtonProps) {
+  return (
+    <button
+      key={key}
```

Comment:
What does `key` do?

Reply:
It seems that React needs a unique identifier for array items. See this:
https://stackoverflow.com/questions/28329382/understanding-unique-keys-for-array-children-in-react-js